policyengine_us/parameters/gov/states/de/tax/income/subtractions/exclusions/pension/military_retirement_exclusion_available.yaml

```diff
@@ -0,0 +1,16 @@
+description: Delaware separates pension exclusion cap amount for military retired filers if this is true.
```

Comment:
Though I'm still not sure what this means - could you try plugging some of the code/law/tax form into chatgpt and ask it to suggest a description?
```suggestion
description: Delaware separates the pension exclusion cap amount for military retired filers if this is true.
```